### DISCUSSION TITLE
Welcome to FluentEmail Discussions!

### DISCUSSION THREAD

**Discussion by lukencode** [lukencode/FluentEmail#235] [General]
## 👋 Welcome!
  We’re using Discussions as a place to ask questions organise development of FluentEmail.
  * Ask questions
  * Offer to contribute
  * Share ideas.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leanardos** [comment]
Hi @lukencode, I would like to know how it behaves when an email sends to multiple addresses, and let's say it failed to send email only some of them. What would be the response? Is it going to return some sort of dictionary with the email addresses and errors related to those email addresses or it is going to be either successful or not successful. And is there any mechanism that it tries to send emails again for the addresses it failed to send?

**Comment by Moglten** [comment]
I really wanna know if the fluent email can send an email without any cridentials. if it's right it can, what service you use to do that.
